addons/website_sale/tests/test_website_sale_cart.py

```diff
@@ -4,6 +4,8 @@
 from odoo.addons.website.tools import MockRequest
 from odoo.exceptions import UserError
 from odoo.tests.common import TransactionCase, tagged
+from odoo import Command
```

Comment:
```suggestion
from odoo.fields import Command
```
personal preference to use more targeted import (even if this works ofc)